common/src/main/java/com/microsoft/hydralab/common/entity/center/TestTaskSpec.java

```diff
@@ -46,4 +47,5 @@ public class TestTaskSpec {
     public String testScope;
     public String testSuiteClass;
     public Map<String, List<DeviceAction>> deviceActions;
+    public List<InspectionStrategy> inspectionStrategies;
```

Comment:
Could you share the reason you make the performanceInspection a member of strategy?

Reply:
From my perspective, the inspection strategy was always bounded with a performance inspection. So, I think it's makes the code more concise and easier to understand. Otherwise, we may need another data structure (`Map` or `Pair`) to maintain the relationship between the inspection strategies and performance inspections.

Reply:
Got it, I think this makes sense, clever move! Mark this comment as resolved.